src/app/data-planes/views/DataPlaneInboundSummaryView.vue

```diff
@@ -15,7 +15,13 @@
           >
 
           <h2 class="summary-title">
-            Inbound :{{ route.params.service }}
+            <template v-if="props.gateway">
+              {{ route.params.service }}
+            </template>
+
+            <template v-else>
+              Inbound :{{ route.params.service.replace('localhost_', '') }}
```

Comment:
**Comment**: I needed to change the route parameter from being just the port to `localhost_${port}` so that I could generalize handling the display of the "compressed" stats code blocks in the various summaries: They can now filter and replace using `${route.params.service}.` and no longer have a hard-coded reference to `'localhost_'`.